cloudevents/service.go

```diff
@@ -17,16 +16,19 @@ import (
 	"time"
 
 	cloudevents "github.com/cloudevents/sdk-go/v2"
+	"github.com/rs/zerolog/log"
```

Comment:
why not using the std lib slog https://go.dev/blog/slog ?

Reply:
same as `http`

Reply:
Both are good choices.  The `http` pacakge was prototyped before Go 1.21 when `slog` was introduced, so for now it is more important to be consistent than cutting-edge.  The packages are very similar, with I believe the key point being very fast with zero allocations per logging event.  Though I do prefer the `zerolog` API a bit, I am happy to undertake the task of updating both `http` and `cloudevent` pacakges to use the standard library in a future PR, because we should indeed use the standard library unless there is a good reason not to.